content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,81 +118,140 @@ On your validator:
 
-3. Configure your validator and stock `rippled` servers to run in a [cluster](cluster-rippled-servers.html).
+    To help avoid this issue, you can use proxies in different geophysical locations.
 
-4. In your validator's `rippled.cfg` file, set `[peer_private]` to `1` to prevent your validator's IP address from being forwarded. For more information, see [Private Peers](peer-protocol.html#private-peers).
+_**To connect your validator to the XRP Ledger network using proxies:**_
+
+1. Set up stock `rippled` servers. For more information, see [Install rippled](install-rippled.html).
+
+2. Configure your validator and stock `rippled` servers to run in a [cluster](cluster-rippled-servers.html).
+
+3. In your validator's `rippled.cfg` file, set `[peer_private]` to `1`. This prevents your validator's IP address from being forwarded. For more information, see [Private Peers](peer-protocol.html#private-peers). It also prevents your validator from connecting to servers other than those defined in the `[ips_fixed]` stanza you defined to run your validator in a cluster.
 
     **Warning:** Be sure that you don't publish your validator's IP address in other ways.
 
-5. Configure your validator host machine's firewall to allow the following traffic only:
+4. Configure your validator host machine's firewall to allow the following traffic only:
 
     - Inbound traffic: Only from IP addresses of the stock `rippled` servers in the cluster you configured.
 
     - Outbound traffic: Only to the IP addresses of the stock `rippled` servers in the cluster you configured and to <https://vl.ripple.com> through port 443.
 
-6. Restart `rippled`.
+5. Restart `rippled`.
 
         $ sudo systemctl restart rippled.service
 
-7. Use the [Peer Crawler](peer-crawler.html) endpoint on one of your stock `rippled` servers. The response should not include your validator. This verifies that your validator's `[peer_private]` configuration is working. One of the effects of enabling `[peer_private]` on your validator is that your validator's peers do not include it in their Peer Crawler results.
+6. Use the [Peer Crawler](peer-crawler.html) endpoint on one of your stock `rippled` servers. The response should not include your validator. This verifies that your validator's `[peer_private]` configuration is working. One of the effects of enabling `[peer_private]` on your validator is that your validator's peers do not include it in their Peer Crawler results.
+
+<!-- { TODO: Future: add a recommended network architecture diagram to represent the proxy, clustering, and firewall setup: https://ripplelabs.atlassian.net/browse/DOC-2046 }-->
+
+
+### Connect using public hubs
+
+This configuration connects your validator to the network using two public hubs. This configuration is similar to [connecting using proxies you run yourself](#connect-using-proxies), but instead you connect through public hubs. Successful public hubs embody the following traits:
```

Comment:
The "successful public hubs" part feels a little out of place. Maybe it could move to `rippled-server-modes.md` as a sub-section of "Reasons to Run a Stock Server" like this?

> ### Public Hubs
> 
> A public hub is a stock server with lots of [peer protocol connections]\(peer-protocol.html) to other servers. You can help the XRP Ledger network maintain efficient connectivity by running a stock server as a public hub. Successful public hubs embody the following traits:
>
> - (bullets here)

(And then link to the new section from here rather than going directly on tangent about public hubs)